images/macos/templates/macOS-12.json

```diff
@@ -0,0 +1,228 @@
+                "./provision/configuration/add-network-interface-detection.sh",
+                "./provision/configuration/autologin.sh",
+                "./provision/configuration/disable-auto-updates.sh",
+                "./provision/configuration/screensaver-off.sh",
+                "./provision/configuration/ntpconf.sh",
+                "./provision/configuration/max-files.sh",
+                "./provision/configuration/shell-change.sh"
+            ],
+            "environment_vars": [
+                "PASSWORD={{user `vm_password`}}",
+                "USERNAME={{user `vm_username`}}"
+            ]
+        },
+        {
+            "type": "shell",
+            "execute_command": "chmod +x {{ .Path }}; {{ .Vars }} {{ .Path }}",
+            "scripts": [
+                "./provision/configuration/preimagedata.sh",
+                "./provision/configuration/configure-ssh.sh",
+                "./provision/core/xcode-clt.sh",
+                "./provision/configuration/configure-machine.sh"
+            ],
+            "environment_vars": [
+                "IMAGE_VERSION={{user `build_id`}}",
+                "IMAGE_OS={{user `image_os`}}"
+            ]
+        },
+        {
+            "type": "shell",
+            "execute_command": "chmod +x {{ .Path }}; sudo {{ .Vars }} {{ .Path }}",
+            "script": "./provision/core/reboot.sh",
+            "expect_disconnect": true
+        },
+        {
+            "type": "shell",
+            "execute_command": "chmod +x {{ .Path }}; {{ .Vars }} {{ .Path }}",
+            "pause_before": "30s",
+            "scripts": [
+                "./provision/core/homebrew.sh",
+                "./provision/core/powershell.sh",
+                "./provision/core/dotnet.sh",
+                "./provision/core/python.sh",
+                "./provision/core/azcopy.sh",
+                "./provision/core/openssl.sh",
+                "./provision/core/ruby.sh",
+                "./provision/core/rubygem.sh",
+                "./provision/core/git.sh",
+                "./provision/core/node.sh"
+            ]
+        },
+        {
+            "type": "shell",
+            "execute_command": "chmod +x {{ .Path }}; {{ .Vars }} pwsh -f {{ .Path }}",
+            "script": "./provision/core/xcode.ps1",
+            "environment_vars": [
+                "XCODE_INSTALL_USER={{user `xcode_install_user`}}",
+                "XCODE_INSTALL_PASSWORD={{user `xcode_install_password`}}"
+            ]
+        },
+        {
+            "type": "shell",
+            "execute_command": "chmod +x {{ .Path }}; sudo {{ .Vars }} {{ .Path }}",
+            "script": "./provision/core/reboot.sh",
+            "expect_disconnect": true
+        },
+        {
+            "type": "shell",
+            "execute_command": "chmod +x {{ .Path }}; {{ .Vars }} {{ .Path }}",
+            "scripts": [
+                "./provision/core/commonutils.sh",
+                "./provision/core/golang.sh",
```

Comment:
It looks like, we missed the [safari.sh](https://github.com/actions/virtual-environments/blob/main/images/macos/provision/core/safari.sh) script in this section

Reply:
Done. Thank you.